### DISCUSSION TITLE
Finding a comment with pull request number

### DISCUSSION THREAD

**Discussion by leo-petrucci** [community/community#27034] [Actions]
<p>I’m currently trying to fix an Action that used to work, but is currently broken.</p>
<p>The problem boils down to <code>github.event.number</code> being empty. I previously used <code>github.event.number</code> to find the PR and a comment within it that I could update, but as of some point over the last month <code>github.event.number</code> became blank.</p>
<p>I have tried:</p>
<pre><code class="lang-auto">      - name: Dump Var
        run: echo "test ${{ github.event.number }}"

      # Looks for a comment with &lt;!-- GH BOT --&gt; somewhere in its body.
      - name: Find Comment
        uses: peter-evans/find-comment@v1
        if: success()
        id: fc
        with:
          issue-number: ${{ github.event.number }}
          body-includes: "&lt;!-- GH BOT --&gt;"
</code></pre>
<p>and</p>
<pre><code class="lang-auto">      - name: Dump Var
        run: echo "test ${{ github.event.pull_request.number }}"

      # Looks for a comment with &lt;!-- GH BOT --&gt; somewhere in its body.
      - name: Find Comment
        uses: peter-evans/find-comment@v1
        if: success()
        id: fc
        with:
          issue-number: ${{ github.event.pull_request.number }}
          body-includes: "&lt;!-- GH BOT --&gt;"
</code></pre>
<p>But “Find Comment” fails with:</p>
<pre><code class="lang-auto">Run peter-evans/find-comment@v1
  with:
    body-includes: &lt;!-- GH BOT --&gt;
    token: ***
    repository: creativiii/wordpress-react
    direction: first
  env:
    npm_config_cache: /home/runner/.npm
Error: Not Found
</code></pre>
<p>. “Dump Var” simply prints out:</p>
<pre><code class="lang-auto">Run echo "test "
test
</code></pre>
<p>Any ideas how I can get this to work?</p>

**Comment by otter-computer** [comment]
<p>What event are you triggering the workflow with, is it a <code>pull_request</code> or <code>pull_request_comment</code> event? And are you filtering on a specific event type when triggering them too?</p>

**Comment by leo-petrucci** [comment, marked as answer]
<p>This was the problem, the <code>push</code> event was overwriting the <code>pull_request</code> event, so the value was undefined.</p>
